### PR TITLE
Fix issue that rectangle can't move to clicked destination on navigator

### PR DESCRIPTION
**Pull request by hiroakit**
When i clicked on navigator after rotation, I found this issue.
So, i fixed it.

issue

### CODE REVIEW

**Comment by avandecreme** [comment]
Looks good to me. Thanks!

File: src/navigator.js [hunk 1]
```diff
@@ -372,7 +372,7 @@ $.extend( $.Navigator.prototype, $.EventSource.prototype, $.Viewer.prototype, /*
  */
 function onCanvasClick( event ) {
     if ( event.quick && this.viewer.viewport ) {
-        this.viewer.viewport.panTo( this.viewport.pointFromPixel( event.position ).rotate( -this.viewer.viewport.degrees, this.viewer.viewport.getHomeBounds().getCenter() ) );
+        this.viewer.viewport.panTo( this.viewport.pointFromPixelNoRotate( event.position ).rotate( -this.viewer.viewport.degrees, this.viewer.viewport.getHomeBounds().getCenter() ) );
```
Comment (avandecreme) [inline src/navigator.js]: Actually, it could be simplified to:
```javascript
this.viewer.viewport.panTo(this.viewport.pointFromPixel(event.position));
```

Comment (hiroakit) [inline src/navigator.js]: Thanks!
I accepted your comment to this PR.

**Comment by iangilman** [comment]
Awesome, thank you!

**Comment by hiroakit** [comment]
Thanks!

**Pull request merged**